tests/quick/commands/test_topic.py

```diff
@@ -132,6 +132,33 @@ def test_execute_required_key_value_type_field(self):
         self.assertEqual(cm.exception.code, 2)
         self.assertTrue("Error: the following arguments are required: -k/--key-type, -v/--value-type" in f.getvalue())
 
+    def test_execute_range(self):
+        creation_data = TopicCreationData(
+            write_type=TopicWriteType.MUTABLE, value_schema=self.GATEWAY_SCHEMA, point=False, range_field="testField"
+        )
+
+        args = self.parser.parse_args(
+            [
+                "topic",
+                self.COMMAND,
+                self.NAME,
+                self.KEY_CMD,
+                self.KEY_TYPE,
+                self.VALUE_CMD,
+                self.VALUE_TYPE,
+                self.SCHEMA_CMD,
+                self.SCHEMA,
+                "--no-point",
+                "--range",
```

Comment:
Does this pass? Shouldn't this be `--rangeField`?

Reply:
This is really strange! The test passes with `--range` even if the argument is `--range-field` or `--rangeField`.
If I change it to something like `--range2` it would fail.

Reply:
Here is the explanation:
https://docs.python.org/dev/library/argparse.html#dest